ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/alter_table.md

```diff
@@ -220,6 +220,18 @@ For example, this command disables automatic partitioning of the table:
 ALTER TABLE series SET (AUTO_PARTITIONING_BY_SIZE = DISABLED);
 ```
 
+Using the `ALTER` command, you can also modify the `TTL` parameters of the table. For example, to increase the delay interval for record deletion:
+```sql
+ALTER TABLE table_name 
+SET (TTL = Interval("PT10S") ON column_name);
+```
+
+If you need to remove the `TTL` from a table, you can do so with the following command:
+```sql
+ALTER TABLE table_name
+RESET (TTL);
+```
+
```

Comment:
There's already the same example below on this page:
![Screenshot 2024-06-03 at 07 26 05](https://github.com/ydb-platform/ydb/assets/623861/9adbfc4b-4e12-4848-8a26-d4f32fd7781a)